source/rmarkdown/annelida_compositional_differences.Rmd

```diff
@@ -281,14 +281,44 @@ plot_tsne <- function(plotdata, colour = NULL) {
 ordination <- calc_ordinations(
   tacos_data = tidy_physeq,
   frequency_threshold = 0,
-  rarefy_even_reads = 100000)
+  rarefy_even_reads = 30780)
```

Comment:
Dat zou ik niet doen. Ik heb de functie zo ingesteld dat er sampling _with replacement_ wordt uitgevoerd. Het is dus geen probleem dat dit op een mooi rond getal zoals 100000 staat ingesteld. Het is ook veel logischer als de meeste samples ordes van grootte meer reads hebben om de rarefication niet helemaal naar beneden te trekken naar het minimum aantal reads in een staal na filtering van stalen met extreem weinig reads. Die 100000 komt ongeveer overeen met het 25 percentiel als ik me goed herinner. 
Het is ook niet erg "elegant" om een exact getal te plakken op die "even reads". Je moet denken in ordes van grootte en mag dus gerust afronden op een duizendtallen.

Reply:
ok bedankt voor de info, ik pas dit terug aan